Client/mods/deathmatch/logic/CClientEntity.cpp

```diff
@@ -303,6 +303,22 @@ CLuaArgument* CClientEntity::GetCustomData(const char* szName, bool bInheritData
     return NULL;
 }
 
+CLuaArguments* CClientEntity::GetAllCustomData(CLuaArguments* table)
+{
+    assert(table);
+
+    // Grab it and return a pointer to the variable
+    std::map<std::string, SCustomData>::const_iterator iter = m_pCustomData->IterBegin();
+    for (; iter != m_pCustomData->IterEnd(); iter++)
```

Comment:
Use `for (auto it = m_pCustomData->IterBeing();`.

I recognise this is copied from `CLuaArguments* CElement::GetAllCustomData` (server) but when we unify the code (if ever), we can just select the better version (this one).